core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineOracle.java

```diff
@@ -309,4 +310,10 @@ public String computeBooleanValue(boolean value) {
   public Driver getDriver() {
     return new oracle.jdbc.driver.OracleDriver();
   }
+
+  @Override
+  public String getEscape(LikeExpression likeExpression) {
+    String escape = likeExpression.getEscape();
+    return escape.isEmpty() ? null : escape;
```

Comment:
Is this also needed in `RdbEnginePostgresql`?

Moving this logic to `com.scalar.db.storage.jdbc.RdbEngineStrategy#getEscape` might be an option.

Reply:
No, we always need the escape clause for PostgreSQL (if it's empty, we need to return the empty string as is). But for Oracle and SQLite, we need to remove the escape clause to disable the escape character.